lib/improver/cli/nowcast_accumulate.py

```diff
@@ -0,0 +1,117 @@
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to accumulate input data given advection velocity fields."""
+
+from improver import cli
+
+inputadvection = cli.create_constrained_inputcube_converter(
+    'precipitation_advection_x_velocity', 'precipitation_advection_y_velocity')
+
+
+@cli.clizefy
+@cli.with_output
+def process(input_cube: cli.inputcube, advection_cubes: inputadvection,
+            *oe_cube: cli.inputcube, attributes_dict: cli.inputjson = None,
+            max_lead_time=360, lead_time_interval=15, accumulation_period=15,
+            accumulation_units='m'):
+    """Module to extrapolate and accumulate the weather with 1 min fidelity.
+
+    Args:
+        input_cube (iris.cube.Cube):
+            The input Cube to be processed.
+        advection_cubes (iris.cube.CubeList):
+            Advection cubes of U and V.
+        oe_cube (iris.cube.Cube):
+            Cube containing the orographic enhancement fields. May have data
+            for multiple times in the cube.
+            Default is None.
+        attributes_dict (dict):
+            Dictionary containing the required changes to the attributes.
+            Default is None.
+        max_lead_time (int):
+            Maximum lead time required (mins).
+            Default is 360.
+        lead_time_interval (int):
+            Interval between required lead times (mins).
+            Default is 15.
+        accumulation_period (int):
+            The period over which the accumulation is calculated (mins).
+            Only full accumulation periods will be computed. At lead times
+            that are shorter than the accumulation period, no accumulation
+            output will be produced.
+        accumulation_units (str):
+            Desired units in which the accumulations should be expressed.
+            e.g. 'mm'
+            Default is 'm'.
+
+    Returns:
+        iris.cube.CubeList:
+            New cubes with accumulated data.
+
+    Raises:
+        ValueError:
+            can either use speed_cube and direction_cube or u_cube and v_cube.
+    """
+    from iris import Constraint
+    import numpy as np
+    from iris.cube import CubeList
+    from improver.nowcasting.accumulation import Accumulation
+    from improver.nowcasting.forecasting import CreateExtrapolationForecast
+    from improver.utilities.cube_manipulation import merge_cubes
+    u_cube = advection_cubes.extract(
+        Constraint("precipitation_advection_x_velocity"), True)
+    v_cube = advection_cubes.extract(
+        Constraint("precipitation_advection_y_velocity"), True)
+
+    if not (u_cube and v_cube):
+        raise TypeError(
+            "Neither u_cube or v_cube can be none")
+    oe_cube = merge_cubes(CubeList(oe_cube))
+
+    # The accumulation frequency in minutes.
+    accumulation_fidelity = 1
```

Comment:
Is there a reason for not having this as an optional argument that we pass in?

Reply:
It is meant to be fixed to 1 minute.